unlock-app/src/pages/log.js

```diff
@@ -32,15 +32,18 @@ export const Log = ({ account, network, transactionFeed }) => {
               <LogHeader>Block Number</LogHeader>
               <LogHeader>Lock Name/Address</LogHeader>
               <LogHeader>Type</LogHeader>
-              {transactionFeed.map(tx => (
-                <React.Fragment key={tx.hash}>
-                  <LogElement>{tx.blockNumber}</LogElement>
-                  <Address href={tx.href} target="_blank">
-                    {tx.lock}
-                  </Address>
-                  <Type type={tx.type}>{tx.readableName}</Type>
-                </React.Fragment>
-              ))}
+              {transactionFeed.map(tx => {
+                const { href, readableName, lock } = metadata[tx.hash]
+                return (
+                  <React.Fragment key={tx.hash}>
```

Comment:
just FYI, because it could be a consistency issue, other places in the code that use fragment use the new jsx shorthand `<></>`. I am thinking probably we should refactor those in favor of `<React.Fragment>` but import it as `{ Fragment } from 'react'`. @julien51 what do you think?

Reply:
A factor to consider is that the shorthand syntax does not allow for a `key` prop. So if we do want to refactor to maintain consistency, we should definitely go with your suggestion.